resources/js/beatmap-discussions/main.coffee

```diff
@@ -357,6 +358,10 @@ export class Main extends React.PureComponent
     @cache.groupedBeatmaps ?= BeatmapHelper.group _.values(@beatmaps())
 
 
+  handleNewDiscussionFocus: =>
+    document.documentElement.style.removeProperty '--scroll-padding-top-extra'
```

Comment:
can use a note it's for [this chrome bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1466472)

Reply:
actually forgot to put a comment after figuring out the cutoffs where the scrolling starts 👀